parla/task_runtime.py

```diff
@@ -1417,12 +1455,12 @@ def __init__(self, environments: Collection[TaskEnvironment], n_threads: Optiona
         # Start the scheduler thread (likely to change later)
         self.start()
 
-    @property
-    @lru_cache(maxsize=1)
+    @ property
```

Comment:
should be removed right?

Reply:
oh I'm not sure. I think vscode just added a space here, I'll fix it in the edits.